From d67c8a9c09275e67086dd00c63afbb44e4a2b3a1 Mon Sep 17 00:00:00 2001
From: Cheng Shao <terrorjack@type.dance>
Date: Sat, 11 Jan 2025 06:48:57 +0000
Subject: [PATCH] hadrian: fix bootstrap with 9.12.1

This patch bumps hadrian index-state to fix bootstrap with 9.12.1.

(cherry picked from commit e16eae6548743870e77c2c93527bab1d24cff81c)

Origin: upstream, https://gitlab.haskell.org/ghc/ghc/-/commit/d67c8a9c09275e67086dd00c63afbb44e4a2b3a1
---
 .gitlab/ci.sh         | 2 +-
 hadrian/cabal.project | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/.gitlab/ci.sh b/.gitlab/ci.sh
index f614ae4c2d3e..9c8fd224436a 100755
--- a/.gitlab/ci.sh
+++ b/.gitlab/ci.sh
@@ -8,7 +8,7 @@ set -Eeuo pipefail
 
 # Configuration:
 # N.B. You may want to also update the index-state in hadrian/cabal.project.
-HACKAGE_INDEX_STATE="2024-10-30T22:56:00Z"
+HACKAGE_INDEX_STATE="2025-01-27T17:45:32Z"
 MIN_HAPPY_VERSION="1.20"
 MIN_ALEX_VERSION="3.2.6"
 
diff --git a/hadrian/cabal.project b/hadrian/cabal.project
index a0d5f0c87073..4b1ac64f77a0 100644
--- a/hadrian/cabal.project
+++ b/hadrian/cabal.project
@@ -4,7 +4,7 @@ packages: ./
 
 -- This essentially freezes the build plan for hadrian
 -- It would be wise to keep this up to date with the state set in .gitlab/ci.sh.
-index-state: 2024-10-30T22:56:00Z
+index-state: 2025-01-27T17:45:32Z
 
 -- unordered-containers-0.2.20-r1 requires template-haskell < 2.22
 -- ghc-9.10 has template-haskell-2.22.0.0
